src/identity/components/GlobalHeader/OrgDropdown.tsx

```diff
@@ -62,10 +74,29 @@ export const OrgDropdown: FC<Props> = ({activeOrg, orgsList}) => {
     },
   ]
 
+  if (isFlagEnabled('createDeleteOrgs') && accountType === 'free') {
+    orgMainMenu.push({
+      name: 'Add More Organizations',
+      iconFont: IconFont.CrownSolid_New,
+      href: '/checkout',
+      className: 'upgrade-payg-add-org--button',
+    })
+  }
+
   const sendDropdownClickEvent = () => {
     event(HeaderNavEvent.OrgDropdownClick, multiOrgTag)
   }
 
+  const additionHeaderItems = []
+  if (
```

Comment:
I think this should be `additionalHeaderItems`

Reply:
good catch in typo.. 🙏